src/serverless-openapi-typescript.ts

```diff
@@ -1,11 +1,25 @@
-'use strict';
-const { upperFirst, mergeWith, set, isArray, get, isEmpty } = require('lodash');
-const tsj = require('ts-json-schema-generator');
-const yaml = require('js-yaml');
-const fs = require('fs');
-
-class ServerlessOpenapiTypeScript {
-  constructor(serverless, options) {
+import type Serverless from "serverless";
+import fs from "fs";
+import yaml from "js-yaml";
+import tsj,  {SchemaGenerator} from "ts-json-schema-generator";
+import {upperFirst, mergeWith, set, isArray, get, isEmpty } from "lodash" ;
```

Comment:
would it be more clear to import lodash as 
```import _ from 'lodash';```?

Reply:
why? (But I didn't do it - I just convert old js file to TS file)